package.json

```diff
@@ -10,7 +10,7 @@
     "typecheck": "tsc",
     "check": "npm run check:qiskit-bot && npm run check:patterns-index && npm run check:metadata && npm run check:spelling && npm run check:internal-links && npm run check:orphan-pages && npm run check:fmt",
     "check:metadata": "node -r esbuild-register scripts/js/commands/checkMetadata.ts",
-    "check:spelling": "cspell --relative --no-progress docs/**/*.md* docs/api/**/*.md* --config cspell/cSpell.json",
```

Comment:
`--relative` seems to be the default: https://cspell.org/docs/getting-started/#command-lint--spell-checking. Note that search only shows `--no-relative`, rather than `--relative`.

We now always use `.mdx`, so it's more obvious to configure that.